include/dlaf/sender/transform_mpi.h

```diff
@@ -85,16 +65,16 @@ struct MPICallHelper {
 template <typename F>
 MPICallHelper(F&&) -> MPICallHelper<std::decay_t<F>>;
 
-/// Lazy transformMPI. This does not submit the work and returns a sender.
+/// Lazy transformMPI. Returns a sender that will submit the work passed in
 template <typename F, typename Sender,
           typename = std::enable_if_t<pika::execution::experimental::is_sender_v<Sender>>>
 [[nodiscard]] decltype(auto) transformMPI(F&& f, Sender&& sender) {
-  using dlaf::internal::continues_on;
-  namespace ex = pika::execution::experimental;
-
-  return continues_on(std::forward<Sender>(sender), dlaf::internal::getMPIScheduler()) |
-         ex::then(dlaf::common::internal::ConsumeRvalues{MPICallHelper{std::forward<F>(f)}}) |
-         ex::drop_operation_state();
+  using dlaf::common::internal::ConsumeRvalues;
+  using pika::execution::experimental::drop_operation_state;
+  using pika::mpi::experimental::transform_mpi;
+  return std::forward<Sender>(sender)                                        //
+         | transform_mpi(ConsumeRvalues{MPICallHelper{std::forward<F>(f)}})  //
```

Comment:
Quite the opposite - I want clang-format to do the alignment of the pipe operators for me, - especially on multi line multi pipe operations, so adding an eol comment forces clang-format to keep each sender on a new line without adding two lines of `// clang-format off` and having to manually line things up myself

Reply:
I agree with John. Using fake comments to give hints to clang-format is better than disable it. In case of a complete reformat `// clang-format off` lines have to be handled manually.